website/docs/module_kitsu.md

```diff
@@ -38,6 +38,18 @@ This functionality cannot deal with all cases and is not error proof, some inter
 openpype_console module kitsu push-to-zou -l me@domain.ext -p my_password
 ```
 
+## Integrate Kitsu Note
+Task status can be automatically set during publish thanks to `Integrate Kitsu Note`. This feature can be configured in:
+
+`Admin -> Studio Settings -> Project Settings -> Kitsu -> Integrate Kitsu Note`.
+
+There are three settings available:
+- `Set status on note` -> turns on and off this integrator.
+- `Note shortname` -> Which status shortname should be set automatically (Case sensitive).
+- `Status exceptions` -> Exceptions for which task status should be left unchanged. You can add as many exceptions as you like. There are two fields to each exceptions: `Equality` (Whether current status should be equal or not equal to the exception status) and `Short name` (Shortname of the exception status).
+
+![Integrate Kitsu Note project settings](assets/integrate_kitsu_note_settings.png)
```

Comment:
Could you have the two examples `Equal` and `Not Equal` in you screenshot please?